controllers/comments.js

```diff
@@ -0,0 +1,74 @@
+const model = require ('../models/comments')
+const userModel = require ('../models/users')
+const todoModel = require ('../models/todo')
+
+module.exports.getAll = function getAll(req, res){
+  model.find({}).exec((err, data) => {
+    if (err) return res.status(404).send()
+    res.json({ data })
+  })
+}
+
+module.exports.post = function post(req, res){
+  const newComment = new model(req.body)
```

Comment:
you should be creating the "new model" after  findOne statement, and in the findOne you can simply find using req.body.userName